src/adapters/supabase/helpers/issues.ts

```diff
@@ -193,4 +193,34 @@ export class Issue extends SuperSupabase {
       return null;
     }
   }
+
+  async findSimilarIssuesToMatch({ markdown, currentId, threshold }: FindSimilarIssuesParams): Promise<IssueSimilaritySearchResult[] | null> {
+    // Create a new issue embedding
+    try {
+      const embedding = await this.context.adapters.voyage.embedding.createEmbedding(markdown);
+      const { data, error } = await this.supabase.rpc("find_similar_issues_to_match", {
+        current_id: currentId,
+        query_embedding: embedding,
+        threshold,
+        top_k: 5,
```

Comment:
Why 5?

Reply:
Probably not but was just curious why you chose 5 but based on brief research it seems appropriate for our usecase.